assets/calc/wrong-type.yml

```diff
@@ -0,0 +1,7 @@
+---
+meta:
+  string: "Hello, World!"
+
+jobs:
+- name: one
+  instances: (( calc "3.1415 * meta.string" ))
```

Comment:
It would be nice to have some explicit tests of nil, arrays, and maps as inputs to the calc operators as well, to make sure we don't introduce any panics